diff -Nru f-spot-0.4.4/configure.in f-spot-0.4.4-dir/configure.in
--- f-spot-0.4.4/configure.in	2008-05-27 23:08:19.000000000 +0200
+++ f-spot-0.4.4-dir/configure.in	2008-05-29 09:29:47.000000000 +0200
@@ -275,7 +275,7 @@
 if test "$have_gs_pkgconfig" = "yes" && test "x$gs_pc_prefix" = "x$gnome_screensaver_prefix"; then
 	GNOME_SCREENSAVER_SAVERDIR=$gs_pc_saverdir
 elif test -d $gnome_screensaver_prefix/lib/gnome-screensaver; then
-	GNOME_SCREENSAVER_SAVERDIR=$gnome_screensaver_prefix/lib/gnome-screensaver/gnome-screensaver
+	GNOME_SCREENSAVER_SAVERDIR=$gnome_screensaver_prefix/lib/gnome-screensaver
 else 
 	GNOME_SCREENSAVER_SAVERDIR=$gnome_screensaver_prefix/libexec/gnome-screensaver
 fi
