--- f-spot-0.4.2/extensions/GalleryExport/Makefile.am.orig	2008-03-12 14:29:47.000000000 +0000
+++ f-spot-0.4.2/extensions/GalleryExport/Makefile.am	2008-03-12 14:30:28.000000000 +0000
@@ -36,7 +36,7 @@
 plugindir = $(pkglibdir)/extensions
 
 install-data-hook:
-	rm -f $(plugindir)/GalleryExport.addin.xml
+	rm -f $(DESTDIR)/$(plugindir)/GalleryExport.addin.xml
 
 plugin_DATA =			\
 	$(PLUGIN_ASSEMBLY)
