diff -urN f-spot-0.3.5.orig/google-sharp/PicasaWeb.cs f-spot-0.3.5/google-sharp/PicasaWeb.cs
--- f-spot-0.3.5.orig/google-sharp/PicasaWeb.cs	2007-03-18 10:33:25.009401990 +0100
+++ f-spot-0.3.5/google-sharp/PicasaWeb.cs	2007-03-18 10:37:35.588846903 +0100
@@ -260,7 +260,8 @@
 			XmlNamespaceManager nsmgr = new XmlNamespaceManager (doc.NameTable);
 			nsmgr.AddNamespace ("photo", "http://www.pheed.com/pheed/");
 			nsmgr.AddNamespace ("media", "http://search.yahoo.com/mrss/");
-			nsmgr.AddNamespace ("gphoto", "http://picasaweb.google.com/lh/picasaweb");
+			nsmgr.AddNamespace ("gphoto", "http://schemas.google.com/photos/2007");
+			nsmgr.AddNamespace ("batch", "http://schemas.google.com/gdata/batch");
 			node = channel.SelectSingleNode ("gphoto:user", nsmgr);
 			user = node.InnerText;
 			node = channel.SelectSingleNode ("gphoto:nickname", nsmgr);
