--- f-spot-0.4.2/src/Makefile.am.orig	2008-04-13 01:03:25.325063605 +0200
+++ f-spot-0.4.2/src/Makefile.am	2008-04-13 01:03:36.824273289 +0200
@@ -316,6 +316,7 @@
 	$(GCONF_PKG)				\
 	$(GTKHTML_SHARP_PKG)			\
 	-pkg:glade-sharp-2.0			\
+	-pkg:gnome-sharp-2.0			\
 	-pkg:gnome-vfs-sharp-2.0
 
 F_SPOT_DISTRESOURCES =			\
diff -Nur f-spot-0.4.4/Makefile.am f-spot-0.4.4-install/Makefile.am
--- f-spot-0.4.4/Makefile.am	2008-05-28 21:37:22.000000000 +0200
+++ f-spot-0.4.4-install/Makefile.am	2008-05-29 22:31:24.000000000 +0200
@@ -89,8 +89,8 @@
 install-data-hook: 
 	$(UPDATE_DESKTOP)
 	if [ "x$(DIR_GIO)" != "x" ] ; then \
-		$(install_sh_DATA) $(top_builddir)/gio-sharp/gio/gio-sharp.dll $(libdir)/f-spot/; \
-		$(install_sh_DATA) $(top_builddir)/gio-sharp/gio/gio-sharp.dll.config $(libdir)/f-spot/; \
+		$(install_sh_DATA) $(top_builddir)/gio-sharp/gio/gio-sharp.dll $(DESTDIR)/$(libdir)/f-spot/; \
+		$(install_sh_DATA) $(top_builddir)/gio-sharp/gio/gio-sharp.dll.config $(DESTDIR)/$(libdir)/f-spot/; \
 	fi
 
 uninstall-hook: 
