--- trunk/src/MainWindow.cs	2008/11/26 08:58:19	4625
+++ trunk/src/MainWindow.cs	2008/12/30 21:53:53	4681
@@ -161,6 +161,8 @@
 	ModeType view_mode;
 	bool write_metadata = false;
 
+	Gdk.Cursor watch = new Gdk.Cursor (Gdk.CursorType.Watch);
+
 	// Tag Icon Sizes
 	public int TagsIconSize {
 		get { return (int) Tag.TagIconSize; }
@@ -1535,7 +1537,7 @@
 
 	void HandlePhotoViewUpdateStarted (PhotoView sender)
 	{
-		main_window.GdkWindow.Cursor = new Gdk.Cursor (Gdk.CursorType.Watch);
+		main_window.GdkWindow.Cursor = watch;
 		// FIXME: use gdk_display_flush() when available
 		main_window.GdkWindow.Display.Sync ();
 	}
@@ -2822,7 +2824,7 @@
 
 	public void UpdateQuery ()
 	{
-		main_window.GdkWindow.Cursor = new Gdk.Cursor (Gdk.CursorType.Watch);
+		main_window.GdkWindow.Cursor = watch;
 		main_window.GdkWindow.Display.Sync ();
 		query.RequestReload ();
 		main_window.GdkWindow.Cursor = null;

