--- configure.in~	2006-03-13 15:51:10.000000000 +0100
+++ configure.in	2006-03-28 21:38:27.195995000 +0200
@@ -125,7 +125,7 @@
 fi
 
 if test -d $gnome_screensaver_prefix/lib/gnome-screensaver; then
-	GNOME_SCREENSAVER_SAVERDIR=$gnome_screensaver_prefix/lib/gnome-screensaver/gnome-screensaver
+	GNOME_SCREENSAVER_SAVERDIR=$gnome_screensaver_prefix/lib/gnome-screensaver
 else 
 	GNOME_SCREENSAVER_SAVERDIR=$gnome_screensaver_prefix/libexec/gnome-screensaver
 fi
