--- f-spot-0.4.2/configure.in.orig	2008-04-12 02:13:35.401587871 +0200
+++ f-spot-0.4.2/configure.in	2008-04-12 02:16:06.391210696 +0200
@@ -74,7 +74,7 @@
 AC_SUBST(F_LIBS)
 
 PKG_CHECK_MODULES(GNOME_SHARP, gnome-sharp-2.0 >= 2.8)
-if pkg-config --atleast-version=2.19.90 gnome-sharp; then
+if pkg-config --atleast-version=2.19.90 gnome-sharp-2.0; then
 	PKG_CHECK_MODULES(GTKHTML_SHARP, gtkhtml-sharp-3.14 >= 2.19.90)
 	GTKHTML_SHARP_PKG="-pkg:gtkhtml-sharp-3.14"
 else
