--- ./src/FolderExport.cs.orig	2006-03-27 11:34:13.000000000 +0200
+++ ./src/FolderExport.cs	2006-03-27 11:34:59.000000000 +0200
@@ -275,6 +275,7 @@
 		protected bool scale;
 		protected int size;
 		protected string description;
+		protected string language;
 		protected System.Uri destination;
 
 		protected ScaleRequest [] requests;
@@ -421,6 +422,14 @@
 				description = value;
 			}
 		}
+
+		public string Language {
+			get {
+				if (language == null)
+					language=GetLanguage();
+				return language;
+			}
+		}
 		
 		public Uri Destination {
 			get {
@@ -437,6 +446,24 @@
 			requests [0].Width = size;
 			requests [0].Height = size;
 		}
+
+		private string GetLanguage()
+		{
+			string language;
+ 
+			if ((language = Environment.GetEnvironmentVariable ("LC_ALL")) == null)
+				if ((language = Environment.GetEnvironmentVariable ("LC_MESSAGES")) == null)
+					if ((language = Environment.GetEnvironmentVariable ("LANG")) == null)
+						language = "en";
+ 
+			if (language.IndexOf('.') >= 0)
+				language = language.Substring(0,language.IndexOf('.'));
+			if (language.IndexOf('@') >= 0)
+				language = language.Substring(0,language.IndexOf('.'));
+			language = language.Replace('_','-');
+ 
+			return language;
+		}
 	}
 
 	class OriginalGallery : FolderGallery
@@ -669,6 +696,8 @@
 			
 			//writer.Write ("<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01 Transitional//EN\">");
 			writer.WriteLine ("<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Strict//EN\" \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd\">");
+			writer.AddAttribute ("xmlns", "http://www.w3.org/1999/xhtml");
+			writer.AddAttribute ("xml:lang", this.Language);
 			writer.RenderBeginTag ("html");
 			
 			WriteHeader (writer);
@@ -781,6 +810,10 @@
 		public void WriteHeader (System.Web.UI.HtmlTextWriter writer)
 		{
 			writer.RenderBeginTag ("head");
+			/* It seems HtmlTextWriter always uses UTF-8, unless told otherwise */
+			writer.Write ("<meta http-equiv=\"Content-Type\" content=\"");
+			writer.Write ("text/html; charset=UTF-8\" />");
+			writer.WriteLine ();
 			writer.RenderBeginTag ("title");
 			writer.Write (gallery_name);
 			writer.RenderEndTag ();
@@ -824,6 +857,8 @@
 
 			//writer.Write ("<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01 Transitional//EN\">");
 			writer.WriteLine ("<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Strict//EN\" \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd\">");
+			writer.AddAttribute ("xmlns", "http://www.w3.org/1999/xhtml");
+			writer.AddAttribute ("xml:lang", this.Language);
 			writer.RenderBeginTag ("html");
 			WriteHeader (writer);
 			
