--- f-spot-0.4.2/src/InfoDisplay.cs.orig	2008-04-13 00:57:50.848051448 +0200
+++ f-spot-0.4.2/src/InfoDisplay.cs	2008-04-13 00:57:58.347536026 +0200
@@ -148,7 +148,7 @@
 
 		private void Update ()
 		{
-			Gtk.HTMLStream stream = this.Begin (null, "text/html; charset=utf-8", Gtk.HTMLBeginFlags.Scroll);
+			Gtk.HTMLStream stream = this.Begin (null, "text/html; charset=utf-8", Gtk.HTMLBeginFlags.KeepScroll);
 			
 			string bg = Color (this.Style.Background (Gtk.StateType.Active));
 			string fg = Color (this.Style.Foreground (Gtk.StateType.Active));
