diff -aurN f-spot-0.1.4.orig/f-spot.desktop.in.in f-spot-0.1.4/f-spot.desktop.in.in
--- f-spot-0.1.4.orig/f-spot.desktop.in.in	2005-12-06 05:50:07.000000000 +0100
+++ f-spot-0.1.4/f-spot.desktop.in.in	2005-12-07 15:15:47.000000000 +0100
@@ -9,7 +9,7 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=Application;Graphics;Photograph;
+Categories=GTK;GNOME;Graphics;Photograph;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=f-spot
 X-GNOME-Bugzilla-Component=General
diff -aurN f-spot-0.1.4.orig/f-spot-view.desktop.in f-spot-0.1.4/f-spot-view.desktop.in
--- f-spot-0.1.4.orig/f-spot-view.desktop.in	2005-09-19 23:46:40.000000000 +0200
+++ f-spot-0.1.4/f-spot-view.desktop.in	2005-12-07 15:16:13.000000000 +0100
@@ -10,7 +10,7 @@
 StartupNotify=false
 Terminal=false
 Type=Application
-Categories=Application;Graphics;Viewer;
+Categories=GTK;GONME;Graphics;Viewer;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=f-spot
 X-GNOME-Bugzilla-Component=General
